.circleci/config.yml

```diff
@@ -0,0 +1,87 @@
+version: 2.1
```

Comment:
I don't think we need this at all, especially since we're not running anything here at the moment . drone should cover the rest.